deploy/csi-azuredisk-controller.yaml

```diff
@@ -37,6 +37,13 @@ spec:
           volumeMounts:
             - mountPath: /csi
               name: socket-dir
+          resources:
+            limits:
+              cpu: 2m
```

Comment:
what about increase limit as 10*request.cpu/memory? 2 times may be too slow

Reply:
2m is too small, I wonder it won't start itself with so small limits. Let's change it to 200m

Reply:
Fixed.